lambdas/replay-cimit-vcs/src/main/java/uk/gov/di/ipv/core/replaycimitvcs/ReplayCimitVcsHandler.java

```diff
@@ -0,0 +1,83 @@
+package uk.gov.di.ipv.core.replaycimitvcs;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestStreamHandler;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import software.amazon.lambda.powertools.logging.Logging;
+import software.amazon.lambda.powertools.tracing.Tracing;
+import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
+import uk.gov.di.ipv.core.library.cimit.exception.CiPostMitigationsException;
+import uk.gov.di.ipv.core.library.domain.ReplayItem;
+import uk.gov.di.ipv.core.library.domain.ReplayRequest;
+import uk.gov.di.ipv.core.library.exceptions.FailedVcReplayException;
+import uk.gov.di.ipv.core.library.helpers.LogHelper;
+import uk.gov.di.ipv.core.library.persistence.item.VcStoreItem;
+import uk.gov.di.ipv.core.library.service.CiMitService;
+import uk.gov.di.ipv.core.library.service.ConfigService;
+import uk.gov.di.ipv.core.library.verifiablecredential.service.VerifiableCredentialService;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReplayCimitVcsHandler implements RequestStreamHandler {
+    private static final Logger LOGGER = LogManager.getLogger();
+    private final ConfigService configService;
+    private final CiMitService ciMitService;
+    private final VerifiableCredentialService verifiableCredentialService;
+
+    @SuppressWarnings("unused") // Used by AWS
+    public ReplayCimitVcsHandler(
+            ConfigService configService,
+            CiMitService ciMitService,
+            VerifiableCredentialService verifiableCredentialService) {
+        this.configService = configService;
+        this.ciMitService = ciMitService;
+        this.verifiableCredentialService = verifiableCredentialService;
+    }
+
+    @SuppressWarnings("unused") // Used through dependency injection
+    @ExcludeFromGeneratedCoverageReport
+    public ReplayCimitVcsHandler() {
+        this.configService = new ConfigService();
+        this.ciMitService = new CiMitService(configService);
+        this.verifiableCredentialService = new VerifiableCredentialService(configService);
+    }
+
+    @Override
+    @Tracing
+    @Logging(clearState = true)
+    public void handleRequest(InputStream inputStream, OutputStream outputStream, Context context) {
```

Comment:
Is there a reason we're using the `InputStream` signature here, rather than allowing AWS to deserialize into the `ReplayRequest` type for us?

Reply:
I had a go at doing at using the in built AWS deserializer but it seemed to be falling over when trying to deserialize into the ReplayRequest object - in the interests of time I resorted to using an InputStream

Reply:
Ok - if we pick this up again I'd like to figure that out, but it's probably not worth it now :)